tests/fixtures_regress/tests.py

```diff
@@ -34,6 +36,17 @@
 _cur_dir = os.path.dirname(os.path.abspath(upath(__file__)))
 
 
+def is_ascii(s):
+    return all(ord(c) < 128 for c in s)
+
+
+def skipIfNonASCIIPath(test_func):
+    return unittest.skipIf(
+        not is_ascii(django.__file__) and six.PY2,
```

Comment:
I tested this branch on a non-ascii path `tēst/django` and there's no crash here. Am I missing something?